src/styles/content/tables/table-cell-inactive/styles.less

```diff
@@ -0,0 +1,131 @@
+& when (@table-cell-inactive-enabled) {
+  /* Table */
+
+  .table {
+
+    tr.inactive td,
+    td.inactive {
+      .element-text-style(table-cell-inactive);
```

Comment:
I matched the CNVS format for this. The goal is to remove this from here and merge into CNVS so I think we should leave this as is unless @ashenden suggests otherwise. Thoughts?